robottelo/ui/container.py

```diff
@@ -71,7 +71,22 @@ def create(self, resource_name, name, command, parameter_list):
                 {'main_tab_name': 'Environment', 'name': 'TTY', 'value': True},
             ]
         """
-        self.assign_value(locators['container.resource_name'], resource_name)
+        # send_keys() can't send left parenthesis (see SeleniumHQ/selenium#674)
```

Comment:
This approach however emulates user behavior in pretty good way - it's typing CR name till that parenthesis part (e.g. `test`) and then selects desired CR (e.g. `test (Docker)`) from the filtered list (lets say 2 items are there - `test (Docker)` and `test (Libvirt)`). As a user i would probably do something similar as i'm too lazy to type full name with all the brackets and usually stop typing when 2-3 results are available :)